rosdep/base.yaml

```
@@ -5647,6 +5647,11 @@ tmux:
   fedora: [tmux]
   gentoo: [app-misc/tmux]
   ubuntu: [tmux]
+trang:
+  debian: [trang]
+  fedora: [trang]
+  gentoo: [trang]
+  ubuntu: [trang]
```

> Re: CentOS @cottsay would know best

There is no official web database for RHEL or CentOS packages. http://pkgs.org is the best unofficial database I have found.

This package is part of the CentOS 7 base repository, but is not packaged for CentOS 8.

http://mirror.centos.org/centos/7/os/x86_64/Packages/trang-20091111-14.el7.noarch.rpm

```suggestion
  rhel:
    '7': [trang]
  ubuntu: [trang]
```

---

silly question then: why put a package key in base.yaml file if we don't know which repo the user is going to try to resolve that key against?

---

I don't understand the question. CentOS Base is the main OS repository. Like Ubuntu main.

---

@cottsay Thanks. I misunderstood - By "no official web database", I thought you meant there was no standard CentOS repository, when what you meant was there's no fancy website to search that repo.